tox.ini

```diff
@@ -14,7 +14,12 @@ deps =
 commands =
     py.test {posargs}
     coverage xml
-    codecov
-    pylint edx_sga
+#    codecov
+#    pylint edx_sga
 
 
+[gh-actions]
```

Comment:
I don't trust this plugin. Tox is great, its plugins aren't always easy to work with on local development environments.